PullUpController/PullUpController.swift

```diff
@@ -478,7 +478,8 @@ extension UIViewController {
      */
     open func addPullUpController(_ pullUpController: PullUpController,
```

Comment:
Can you please add the documentation for the `completion` parameter as well? Same thing for the `removePullUpController` method.

Reply:
oh yeah, oops!